.go-version

```diff
@@ -1 +1 @@
-1.15.9
+1.16.3
```

Comment:
We found an ugly error while downloading a dependency of Go's cobra on CI, with Go 1.15: See https://github.com/spf13/viper/issues/1161